include/frontend/assembler/op-with-info.hpp

```diff
@@ -5,13 +5,14 @@
 #include "ChaiVM/interpreter/autogen/operations.hpp"
 #include "ChaiVM/interpreter/instruction.hpp"
 
-/*
- * @todo #41:90min Rename this class
+/**
+ * Class that encapsulates chai::interpreter::Operation and provides some
+ *  useful methods.
  */
-class OpString {
+class OperationWithInfo {
```

Comment:
@c71n93 To be honest, I don't like this name

Reply:
@levBagryansky maybe, but it is obviously better, than previous. If you have any ideas, you are welcome.

Reply:
@c71n93 OpString was not the best option but it was clearer because this class is `Operation` and provides methods with `string`

Reply:
@levBagryansky done